components/scream/src/physics/rrtmgp/tests/CMakeLists.txt

```diff
@@ -1,6 +1,32 @@
 include(ScreamUtils)
 
+# Required libraries
+set (NEED_LIBS rrtmgp scream_control scream_share yakl)
+
+set (SRC rrtmgp_tests.cpp rrtmgp_test_utils.cpp)
+
 # NOTE: tests inside this if statement won't be built in a baselines-only build
 if (NOT ${SCREAM_BASELINES_ONLY})
-  CreateUnitTest(rrtmgp_tests rrtmgp_tests.cpp "rrtmgp;scream_share")
+    set_source_files_properties(${SRC} PROPERTIES CXX_STANDARD 14)
+    set_source_files_properties(${SRC} PROPERTIES COMPILE_FLAGS "${YAKL_CXX_FLAGS}")
+    CreateUnitTest(rrtmgp_tests "${SRC}" "${NEED_LIBS}" LABELS "rrtmgp")
+    set_target_properties(rrtmgp_tests PROPERTIES CXX_STANDARD 14)
+    #target_include_directories(rrtmgp_tests PUBLIC ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/cpp/examples/all-sky)
+    #target_include_directories(rrtmgp_tests PUBLIC ${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/cpp/examples/all-sky)
+    target_include_directories(rrtmgp_tests PUBLIC ${SCREAM_BASE_DIR}/../../externals/YAKL)
 endif()
+
+# Copy RRTMGP absorption coefficient lookup tables to local data directory
+FILE (MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/data)
+CONFIGURE_FILE(${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/rrtmgp/data/rrtmgp-data-sw-g224-2018-12-04.nc
+               ${CMAKE_CURRENT_BINARY_DIR}/data COPYONLY)
+CONFIGURE_FILE(${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/rrtmgp/data/rrtmgp-data-lw-g256-2018-12-04.nc
```

Comment:
Here we have some dependency on files in EAM.  How big are these?  Are they small enough that it would make sense to store them locally with SCREAM?

Reply:
These are actually RRTMGP files (i.e., included in the external RRTMGP repo that we pull in as a submodule), not EAM files. I'm including them from the EAM directory because I elected to not pull in a separate clone of RRTMGP to the SCREAM directory. I think probably a better thing to do would be to move the RRTMGP submodule to the main "externals" directory.

Reply:
@AaronDonahue are you okay with this? I think moving the RRTMGP submodule up to the main "externals" directory should be an upstream thing, not a SCREAM thing, if we decide to do that.

Reply:
Yes, I'm okay with this setup.  As long as there is an explanation.  I guess we would expect that at some point in the future we will have a similar set up in scream.  My understanding is that long-term we don't want scream to be dependent on anything in eam.  But for now this should be fine.  Maybe we can add a TODO comment or a git issue so we don't lose sight of this dependency.

Reply:
For now I just added a TODO comment. I will put moving the entire RRTMGP code to the top level externals directory on my list of things to do upstream. Alternatively I guess we could have an additional copy of the submodule in SCREAM? That seems silly though.